### ISSUE TITLE
Name of the video and the date is black and I can't click on the channel

### ISSUE THREAD

**Issue by GitAcc53** [ynshung/better-yt-shorts#216]
This problem started today. I tried in both Chrome and Brave.

Also another problem is, when I click a short, the video is very small at first and 1 second later video becomes normal sized. This wasn't a thing a few months ago.

**Comment by ynshung** [comment]
Thanks for the report, can you help test v3.6.0 once it's updated on your side?

**Comment by GitAcc53** [comment]
> Thanks for the report, can you help test v3.6.0 once it's updated on your side?

Problem is fixed. But this problem "when I click a short, the video is very small at first and 1 second later video becomes normal sized" is not fixed.

**Comment by ynshung** [comment]
> > Thanks for the report, can you help test v3.6.0 once it's updated on your side?
> 
> Problem is fixed. But this problem "when I click a short, the video is very small at first and 1 second later video becomes normal sized" is not fixed.

Would you mind testing to see if the issue only occurs when the extension is enabled?

**Comment by GitAcc53** [comment]
Yeah the issue only occurs when the extension is enabled. I tried in both Chrome and Brave.

**Comment by Luke0094** [comment]
Never tought it was a problem but probably he mean this:
<img width="835" alt="image" src="https://github.com/ynshung/better-yt-shorts/assets/57552558/632bb9ae-ab74-4839-8cdd-8da95d9e4530">
 Is not really an issue and original size will load just after the loading.
